release-notes/con-relnotes-fixed-issues.adoc

```diff
@@ -1,15 +1,52 @@
 [id='con-relnotes-fixed-issues_{context}']
 = Fixed issues
 
-== Security issues resolved in {product} 1.1.0 and 1.1.1
+== Fixed security issues
 
-=== Security fixes in {product} 1.1.1
+=== Fixed security issues in {product} 1.1.5
 
-A number of security fixes have been applied in this release, including the following:
+This section lists fixed security issues with {product} 1.1.5:
+
+link:https://access.redhat.com/security/cve/CVE-2024-29041[CVE-2024-29041]::
++
+--
+A flaw was found in the Express.js minimalist web framework for node. Versions of Express.js before 4.19.0 and all pre-release alpha and beta versions of 5.0 are affected by an open redirect vulnerability using malformed URLs. When a user of Express performs a redirect using a user-provided URL, Express performs an encode using encodeurl on the contents before passing it to the location header. This issue can cause malformed URLs to be evaluated in unexpected ways by common redirect allow list implementations in Express applications, leading to an Open Redirect via bypass of a properly implemented allow list. The main method impacted is res.location(), but this is also called from within res.redirect(). The vulnerability is fixed in 4.19.2 and 5.0.0-beta.3.
+--
+
+=== Fixed security issues in {product} 1.1.1
+
+This section lists fixed security issues with {product} 1.1.1:
+
+link:https://access.redhat.com/security/cve/CVE-2024-21892[CVE-2024-21892]::
++
+--
+Code injection and privilege escalation through Linux capabilities (nodejs)
+--
+
+link:https://access.redhat.com/security/cve/CVE-2024-22019[CVE-2024-22019]::
++
+--
+Reading unprocessed HTTP request with unbounded chunk extension allows DoS attacks (nodejs)
+--
+
+link:https://access.redhat.com/security/cve/CVE-2023-46809[CVE-2023-46809]::
++
+--
+Vulnerable to timing variant of the Bleichenbacher attack against PKCS#1 v1.5 padding (Marvin) (nodejs)
+--
+
+link:https://access.redhat.com/security/cve/CVE-2024-28176[CVE-2024-28176]::
++
+--
+Jose was found to have an uncontrolled resource consumption vulnerability. Under certain conditions, the user's environment can consume an unreasonable amount of CPU time or memory during JWE decryption operations, leading to a denial of service. This affected the ocm-backend plugin. Jose updated to 4.15.5.
+--
+
+link:https://access.redhat.com/security/cve/CVE-2024-28849[CVE-2024-28849]::
++
+--
+A vulnerability was found in the follow-redirects package. While processing the cross-domain redirection, follow-redirects clears authorization headers, however, it misses clearing proxy-authentication headers, which contain credentials as well. This issue may lead to credential leaking, having a high impact on data confidentiality. Updated to 1.15.6.
+--
 
-* link:https://errata.devel.redhat.com/advisory/128590[NodeJS update for RHSA 128590] 
```

Comment:
I've replaced this link which can't be seen by customers with the list of 3 CVEs included in that errata. I figure that's way better customer experience. :D